tasks/jasmine.js

```diff
@@ -194,7 +194,7 @@ module.exports = function(grunt) {
 
     phantomjs.on('console', function(msg) {
       thisRun.cleanConsole = false;
-      if (options.display === 'full') {
+      if (options.display === 'full' || options.showLogging) {
```

Comment:
I alluded in the first comment that `showLogging = undefined` could be explicitly handled to provide backwards compatibility.

```js
var fullLogging = (typeof(options.showLogging) == 'undefined' && options.display == 'full');
if (options.showLogging || fullLogging) {
```